rust/private/rustc.bzl

```diff
@@ -580,7 +580,7 @@ def rustc_compile_action(
     if hasattr(ctx.attr, "out_binary"):
         out_binary = getattr(ctx.attr, "out_binary")
 
-    return establish_cc_info(ctx, crate_info, toolchain, cc_toolchain, feature_configuration) + [
+    info = [
```

Comment:
Nit: rename `info` to `providers` (I could be convinced to other plural words).

Reply:
Done!